### PR TITLE
Contrib Theano v.0.6.0

### PR DESCRIPTION
**Pull request by tloizou**
Contributing Theano v 0.6.0 with goolf 1.4.10 and ictce 5.3.0

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/t/Theano/Theano-0.6.0-goolf-1.4.10-Python-2.7.3.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % (python, pythonver)
+
+dependencies = [
+                (python, pythonver),
+               ]
+
+sanity_check_paths = {
+                      'files': ['bin/theano-cache',
+                                'lib/python%s/site-packages/%s-%s-py%s.egg' % (pythonshortver, name, version, pythonshortver)],
```
Comment (ocaisa) [inline easybuild/easyconfigs/t/Theano/Theano-0.6.0-goolf-1.4.10-Python-2.7.3.eb]: This line is too long...120 character limit

File: easybuild/easyconfigs/t/Theano/Theano-0.6.0-ictce-5.3.0-Python-2.7.3.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % (python, pythonver)
+
+dependencies = [
+                (python, pythonver),
+               ]
+
+sanity_check_paths = {
+                      'files': ['bin/theano-cache',
+                                'lib/python%s/site-packages/%s-%s-py%s.egg' % (pythonshortver, name, version, pythonshortver)],
```
Comment (ocaisa) [inline easybuild/easyconfigs/t/Theano/Theano-0.6.0-ictce-5.3.0-Python-2.7.3.eb]: This line is too long...120 character limit

**Comment by tloizou** [comment]
@boegel: Style fixes done. Ready for testing

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2bbde1522c175ae0327e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @tloizou!

**Pull request merged**